code/modules/mob/living/carbon/human/species_types/roguetown/fey/_fey.dm

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2025 Jerry Goldman
+ * Discord: "___.__.__.__.__.___"
+ *
+ * DUAL LICENSE NOTICE (AGPLv3-compliant):
+ *
+ * This file is made available under TWO possible licenses. You must
+ * choose ONE of them. You do NOT receive both simultaneously.
+ *
+ * ------------------------------------------------------------------
+ * 1) GNU AGPLv3 LICENSE (applies ONLY within Ratwood-Vale/Ratwood-2.0)
+ * ------------------------------------------------------------------
+ * When this file is used as part of the Ratwood-Vale/Ratwood-2.0
+ * project, it is licensed under the GNU Affero General Public License v3.0.
+ *
+ * This includes the AGPLv3 network clause:
+ *  - If this file is used in software that is made available to users
+ *    over a network (e.g. a game server), the complete corresponding
+ *    source code must be provided to those users, as required by AGPLv3.
+ *
+ * ------------------------------------------------------------------
+ * 2) PROPRIETARY LICENSE (for ANY other project or repository)
+ * ------------------------------------------------------------------
+ * Outside of Ratwood-Vale/Ratwood-2.0, this file is NOT distributed
+ * under the AGPLv3. Instead, it is provided solely under a proprietary
+ * license. Explicit written permission from the copyright holder is
+ * required for any use, modification, or redistribution outside of
+ * the Ratwood-Vale/Ratwood-2.0 project.
+ *
+ * ------------------------------------------------------------------
+ * WHY THIS IS AGPLv3-COMPLIANT AND NOT A "RESTRICTION"
+ * ------------------------------------------------------------------
+ *
+ * This is a standard and fully compliant dual-licensing model used by
+ * numerous open source projects (e.g. MySQL, Qt, etc.).
```

Comment:
incorrect. you cannot stipulate "outside of [X]" in your dual licensing scheme. if it is AGPLv3 code then it is AGPLv3 code, that's the nature of copyleft. you are permitted to dual license it, but all that would do is allow a closed source codebase with no AGPLv3 code to use your code.